### DISCUSSION TITLE
Portuguese (Brazil) Translation

### DISCUSSION THREAD

**Discussion by PedroRangel7** [Romanitho/Winget-AutoUpdate#237] [General]
Hello!
I've translated toast notifications to Portuguese - Brazil (pt-br):
[pt-br.zip](https://github.com/Romanitho/Winget-AutoUpdate/files/10050143/pt-br.zip)
How do I get Winget-AutoUpdate to use it?
Thanks!

**Comment by KnifMelti** [comment]
Hello, @PedroRangel7!
In your OS, test via Powershell `(Get-Culture).Parent` if the OS is set in the language you've translated to.
This is the way WAU checks the locale (my **Swedish** Windows):
![image](https://user-images.githubusercontent.com/102996177/202918177-d7abfd00-7f5f-4a9b-b85f-e796d900e7f3.png)
Rename your file to **Name**.xml (in my case **sv**.xml)
Put the file under `C:\ProgramData\Winget-AutoUpdate\locale` (or where you installed WAU to)

**Reply by Romanitho** [reply]
Exactly !

**Comment by thiagotw** [comment]
> Hello! I've translated toast notifications to Portuguese - Brazil (pt-br): [pt-br.zip](https://github.com/Romanitho/Winget-AutoUpdate/files/10050143/pt-br.zip) How do I get Winget-AutoUpdate to use it? Thanks!

I just did the same.... to change it take look in my pic 

![Captura de tela 2022-11-30 150022](https://user-images.githubusercontent.com/27924949/204873567-69e1393a-5656-4d5c-afe1-09e06cf728b3.png)

My File
[br.zip](https://github.com/Romanitho/Winget-AutoUpdate/files/10125975/br.zip)

@Romanitho would be great if we can select the language on your another project [Winget-Install-GUI](https://github.com/Romanitho/Winget-Install-GUI)

Thanks

**Reply by KnifMelti** [reply]
You don't have to change the code in `Get-NotifLocale`...
...it should still be (if (Get-Culture).Parent = BR (https://learn.microsoft.com/en-us/bingmaps/rest-services/common-parameters-and-types/supported-culture-codes)):
```
    #Set English if file doesn't exist
    else {
        $LocaleDisplayName = "English"
        $LocaleFile = "$WorkingDir\locale\en.xml"
    }

```

**Reply by Romanitho** [reply]
to avoid a lot of xml files, we decided to use only parent language. As @KnifMelti said, try to check what your computer returns when you run this powershell command:  
`((Get-Culture).Parent).name`
for me, it is `fr` for instance
